src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -82,6 +82,20 @@ static size_t get_post_ops_count(const program_node& node) {
     return onednn_post_ops_count;
 }
 
+static bool is_reduce_blocked_axes(reduce_node const& node) {
+    auto prim = node.get_primitive();
+    auto reduce_axes = prim->axes;
+    auto input_layout = node.input().get_output_layout();
+
+    if (prim->keep_dims == false &&
+        (find(reduce_axes.begin(), reduce_axes.end(), 1) != reduce_axes.end() ||
```

Comment:
How about using the count function?
```suggestion
        (count(reduce_axes.begin(), reduce_axes.end(), 1) > 0 ||
```

Reply:
Applied